contribs/gmf/src/services/permalink.js

```diff
@@ -645,6 +651,12 @@ gmf.Permalink.prototype.handleLayersRemove_ = function(evt) {
  */
 gmf.Permalink.prototype.registerLayer_ = function(layer, opt_init) {
 
+  if (layer instanceof ol.layer.Group) {
+    layer.getLayers().forEach(function(layer) {
+      this.registerLayer_(layer, opt_init);
+    }, this)
+  }
+
```

Comment:
Ok @adube, I talked with @fgravin about it. You're right, there is no need to put a listener on ol.layer.Group objects. I going to fix it on a new PR right now. I'll let you know when it's done.